airbyte-integrations/bases/source-acceptance-test/unit_tests/test_core.py

```diff
@@ -300,3 +300,685 @@ def test_validate_oauth_flow(connector_spec, expected_error):
             t.test_oauth_flow_parameters(connector_spec)
     else:
         t.test_oauth_flow_parameters(connector_spec)
+
+
+@pytest.fixture
+def records():
+    test1_records = [
```

Comment:
This test is to verbose and very hard to figure out if something would fail. Could you come up with minimal data representig the  case like:
```python
[
{a1: 1, a2: 1 },
{a2: 1, a3: 1 }
]
```
and so on.  Case should be easy to understand on first glance.